Firmware/RTK_Everywhere/System.ino

```diff
@@ -365,14 +365,20 @@ void printReports()
             // If we are in rover mode, display HPA and SIV
             if (inRoverMode() == true)
             {
+                systemPrintf("Rover: ");
```

Comment:
So what gets printed if hpa >= 30? It looks like dead space to me. And else { ">30"} would suffice?

Reply:
Yeah. If hpa was > 30m, the accuracy reporting was skipped...

The feet and inches complicate matters somewhat.

The latest commit takes this to its logical conclusion. The scale table now contains two scales, "m" and "m", or "ft" and "in", with a defined point at which to change from one to the other. And a limit for accuracy reporting: 30.0m or 100.0ft. It also means the table can be alphabetised if desired... And would make it easy to add "m" and "cm" or "mm" in future.